static/app/views/organizationCreate/index.tsx

```diff
@@ -119,18 +125,35 @@ function OrganizationCreate() {
             />
           )}
           {termsUrl && privacyUrl && (
+            <TermsWrapper hasDataConsent={hasDataConsent}>
+              <CheckboxField
+                name="agreeTerms"
+                label={tct(
+                  'I agree to the [termsLink:Terms of Service] and the [privacyLink:Privacy Policy]',
+                  {
+                    termsLink: <a href={termsUrl} />,
+                    privacyLink: <a href={privacyUrl} />,
+                  }
+                )}
+                inline={false}
+                stacked
+                required
+              />
+            </TermsWrapper>
+          )}
+          {hasDataConsent && !isSelfHosted && (
             <CheckboxField
-              name="agreeTerms"
+              name="aggregatedDataConsent"
```

Comment:
It works to use the `isSelfHosted` flag, but would it make more sense for this to be in getsentry?